### PR TITLE
Docs: Update node-status page

### PR DESCRIPTION
**Pull request by brianshumate**
- Add details about Desired Status and Client Status for #264

### CODE REVIEW

File: website/source/docs/commands/node-status.html.md.erb [hunk 1]
```diff
@@ -149,6 +149,25 @@ ed3665f5  8bf94335  example  cache       run             running
...
 
+You will note that in the above examples, the **Allocations** output contains
+columns labeled **Desired Status** and **Client status**.
+
+Desired Status represents the goal of the scheduler on the allocation with
+the following valid statuses:
+
+- *run*: The allocation should run
+- *stop*: The allocation should stop
+- *evict*: The allocation was evicted and should stop
```
Comment (dadgar) [inline website/source/docs/commands/node-status.html.md.erb]: I would remove evict for now since we never put an allocation in that state at the moment.

**Comment by dadgar** [comment]
Looks good!

**Comment by brianshumate** [comment]
Thanks, removed evict!

**Comment by dadgar** [comment]
Thanks Brian!!

**Pull request merged**